test/torch/tensors/test_tensor.py

```diff
@@ -0,0 +1,44 @@
+from unittest import TestCase
+import torch
+
+
+import syft
+from syft.frameworks.torch.tensors import TorchTensor, PointerTensor
+
+
+class TestNative(TestCase):
```

Comment:
I think we switched `TestCase` to object. Pytest does not require `TestCase`, but `TestCase` is disallowing the use of parametrizing methods.